applications/vessel-history/src/features/profile/components/activity/description/ActivityDescriptionEEZ.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+import { selectEezById } from 'features/regions/regions.selectors'
+import { RegionId } from 'features/regions/regions.slice'
+
+interface ActivityDescriptionProps {
+  regionId: RegionId
+  type: 'group' | 'event'
+  ocean?: string
+  count?: number
+}
+
+const ActivityDescriptionEEZ: React.FC<ActivityDescriptionProps> = (props): React.ReactElement => {
+  const regionId = props.regionId
+  const { t } = useTranslation()
+
+  const region = useSelector(selectEezById(regionId))
+  const regionName = region?.label?.replace('Exclusive Economic Zone', 'EEZ') || props.ocean || 'unknown'
```

Comment:
If we need to modify/normalize the labels of the regions I'd try to do that in other place and not in a component before rendering, because it'll only be applied for this component. Instead I'll try to apply those transformations before storing the regions data in the state (could be the thunk) so that's already fixed there and available for the whole app. Makes sense?

Reply:
maybe yes but I am not sure if this replace is needed, I did it to fit with the design